oath/README.md

```diff
@@ -52,23 +52,23 @@ First, create an `OathApplication` to select OATH applet on YubiKey.
     //    } catch (ParseUriException | IOException | ApduException e) {
     //        // handle errors
     //    }
-    
+
 ```
 
 Next, use the `OathApplication` to add, remove, list, and calculate credentials.
 
-The `OathApplication` implements the YKOATH protocol. Refer to the [YKOATH protocol specification](https://developers.yubico.com/OATH/YKOATH_Protocol.html) for more details. 
+The `OathApplication` implements the YKOATH protocol. Refer to the [YKOATH protocol specification](https://developers.yubico.com/OATH/YKOATH_Protocol.html) for more details.
 
-In addition, the `OathApplication` provides an interface for setting/validating a password on credential storage, calculating all credentials and resetting the OATH application to its default state. For the complete list of methods look at the [`OathApplication` class documentation](src/main/java/com/yubico/yubikit/oath/OathApplication.java).  
+In addition, the `OathApplication` provides an interface for setting/validating a password on credential storage, calculating all credentials, and resetting the OATH application to its default state. For the complete list of methods look at the [`OathApplication` class documentation](src/main/java/com/yubico/yubikit/oath/OathApplication.java).  
 
-The **OATH** module also provides a class for defining an OATH `Credential`. Use its convenience initializer `Credential.parseUri` to parse the credential parameters from Uri of [Key Uri Format](https://github.com/google/google-authenticator/wiki/Key-Uri-Format).
+The **OATH** module also provides a class for defining an OATH `Credential`. Use its convenient initializer `Credential.parseUri` to parse the credential parameters from the Uri of the [Key Uri Format](https://github.com/google/google-authenticator/wiki/Key-Uri-Format).
```

Comment:
Change "convenience initializer" to "factory method" which is a more commonly used term.